src/ui/public/filter_bar/filter_bar.less

```diff
@@ -1,7 +1,7 @@
 @import (reference) "~ui/styles/variables";
 
 filter-bar {
-  z-index: 20 !important;
+  z-index: 2 !important;
```

Comment:
:+1: b02b452fe

I added the new file to the variables.less file seeing as how both of those files were already including it... Let me know if you think it'd be better left out.